py-polars/polars/_utils/construction/series.py

```diff
@@ -406,6 +406,7 @@ def pandas_to_pyseries(
     values: pd.Series[Any] | pd.Index[Any] | pd.DatetimeIndex,
     dtype: PolarsDataType | None = None,
     *,
+    strict: bool = True,
```

Comment:
wait, this requires a more general solution - you've added `strict` as an argument but it's not being passed through anywhere

Reply:
How about this time, passing `strict` like `pandas_to_pyseries(name, values, dtype=dtype, strict=strict)`? I'm not entirely sure I understand what you mean.

Since `s` is a `PySeries`, `PySeries.cast()` requires the `strict` positional argument. It seems that `strict` does not have a default value, so it must be provided. 

If we want to avoid passing `strict` directly, another solution could be:

```python
s = wrap_s(arrow_to_pyseries(
    name, plc.pandas_series_to_arrow(values, nan_to_null=nan_to_null)
))
return s._s if dtype is None else s.cast(dtype)._s
```

In the above code, `s` is a `Series`, and `Series.cast()` has a default `strict=True`.

Should we use `Series.cast()`? Or should we just pass `strict` before like now? Or have I missed something else? Perhaps you could provide more explanation.

Reply:
I think what you've done now is fine, but `strict` should probably be inside `arrow_to_pyseries`, and in this function you just pass it through